platform/android/export/export_plugin.cpp

```diff
@@ -1649,7 +1649,6 @@ void EditorExportPlatformAndroid::get_export_options(List<ExportOption> *r_optio
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "one_click_deploy/clear_previous_install"), false));
 
 	r_options->push_back(ExportOption(PropertyInfo(Variant::INT, "version/code", PROPERTY_HINT_RANGE, "1,4096,1,or_greater"), 1));
-	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "version/name"), "1.0"));
```

Comment:
@Calinou I'm wondering if it would be better to keep this property and have it default to `application/config/version`.
The advantage of keeping is that it allows you to customize the version string per export preset (for example, you could have one Android export preset that support VR and another one that doesn't for the same game, and this would allow to update their version name accordingly).